pages/opengraph/collection/[collectionId].tsx

```diff
@@ -0,0 +1,42 @@
+import { useRouter } from 'next/router';
+import { OpenGraphPreview } from 'components/opengraph/OpenGraphPreview';
+import useCollectionById from 'hooks/api/collections/useCollectionById';
+
+export default function OpenGraphCollectionPage() {
```

Comment:
this setup is so freaking cool

Reply:
i thought we'd use raw html/css but the fact that it's a plain nextjs route is amazing